### PR TITLE
deleted unnecessary tests

### PR DESCRIPTION
**Pull request by crmorford**

### CODE REVIEW

**Comment by bluong** [comment]
Quicktests are from my view are QE domain.  If QE feels that these tests are unnecessary, I'm good with it.

File: quicktests/overlaying/tests/basic/grid_plot.js [hunk 1]
```diff
@@ -1,32 +0,0 @@
-
-function makeData() {
-  "use strict";
-
-    var data = [{name: "jon", y: 10, type: "q1"}, {name: "dan", y: 2, type: "q1"}, {name: "zoo", y: 1, type: "q1"}];
```
Comment (jtlan) [inline quicktests/overlaying/tests/basic/grid_plot.js]: This part caught #2014, but I think we should accomplish that by having messy/wrong-type data in other tests.

**Comment by jtlan** [comment]
As @bluong said, you're the master of these, so do what you think is appropriate. I think there's a few key things some of them are testing that we should keep in mind as we build newer tests.

**Comment by aicioara** [comment]
Should we delete them or, instead, just make them easier to follow (have a notion of loading, better comparison highlight, etc)?

**Comment by bluong** [comment]
@aicioara I think the approach that we're going with is along the lines of these tests are not necessarily useful and that even if they are useful, their usefulness is limited and probably should be replaced by better tests.  From what I hear, @crmorford is looking into better tests than the ones being deleted.

**Comment by aicioara** [comment]
@bluong But should we not first implement the solution then delete the obsolete? (Just asking I probably have no context)

**Comment by bluong** [comment]
I would defer to @crmorford here...

I think the point of this pr is remove the extraneous cruft in the quicktests branch.

**Comment by crmorford** [comment]
These aren't catching any bugs. No reason to keep them :(

**Pull request merged**